src/common/icon/getIconAnimation.js

```diff
@@ -0,0 +1,28 @@
+import clsxd from 'clsx-dedupe';
+
+const getIconAnimation = (animate) => {
```

Comment:
One of the shared functions that returns a value that is dependent on one of the icon levers..